bluesky_queueserver/manager/profile_tools.py

```diff
@@ -289,3 +290,52 @@ def load_devices_from_happi(device_names, *, namespace, **kwargs):
     ns_dict = ns.__dict__
     namespace.update(ns_dict)
     return list(ns_dict)
+
+
+_env_re_worker_active = "QSERVER_RE_WORKER_ACTIVE"
+
+
+def set_re_worker_active():
+    """
+    Set the environment variable used to determine if the current process is RE Worker process.
+    Subsequent calls to ``is_re_worker_active`` in the current process will return ``True``.
+    This function should not be used in the startup scripts.
+    """
+    os.environ[_env_re_worker_active] = ""
```

Comment:
Should the value be "True" or "1" instead of an empty string? We may want to do something like the following to reset it:
```bash
$ QSERVER_RE_WORKER_ACTIVE= <our command>
```
and this will enable it:
```bash
$ QSERVER_RE_WORKER_ACTIVE=1 <our command>
```